libs/analytics/src/lib/+state/analytics.service.ts

```diff
@@ -122,4 +148,44 @@ export class AnalyticsService extends BlockframesCollection<Analytics> implement
     const isConcierge = profile?.email.includes('concierge');
     return isBlockframesAdmin || isConcierge || Object.values(centralOrgId).includes(profile?.orgId);
   }
+
+  public async loadAnalyticsData() {
+    if (this.analyticsCache.length) return;
+    const rows = await this.getAnalyticsActiveUsers({});
+    this.analyticsCache = rows.map(r => ({
+      uid: r.user_id,
+      firstConnexion: r.first_connexion.value,
+      lastConnexion: r.last_connexion.value,
+      sessionCount: r.session_count,
+      pageView: r.page_view,
+    }));
+  }
+
+  getLastConnexion(uid: string): Date {
```

Comment:
There is an issue with the typing. The return type should be `Date | undefined`. so you could do : 
```suggestion
  getLastConnexion(uid: string) {
    return this.analyticsCache.find(u => u.uid === uid)?.lastConnexion;
  }
```
Same for below